MAUI/Cartesian-Charts/Annotation.md

```diff
@@ -0,0 +1,659 @@
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+   
+![Line Annotation in MAUI Chart](Annotation_image/Line_annotation.png)
+
+## Vertical and Horizontal line annotation
+
+The [VerticalLineAnnotation]() and [HorizontalLineAnnotation]() are used to draw vertical and horizontal lines at specific points in the chart area.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+        <chart:SfCartesianChart.Annotations>
+            <chart:VerticalLineAnnotation X1="2"/>
+            <chart:HorizontalLineAnnotation Y1="20"/>
+        </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %} 
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    var vertical = new VerticalLineAnnotation()
+    {
+        X1 = 2,
+    };
+
+    var horizontal = new HorizontalLineAnnotation()
+    {
+        Y1 = 20,
+    };
+
+    chart.Annotations.Add(vertical);
+    chart.Annotations.Add(horizontal);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation in MAUI Chart](Annotation_image/Horizontal_vertical_annotation.png)
+
+## Displaying axis label for vertical and horizontal line annotations
+
+The [VerticalLineAnnotation]() and [HorizontalLineAnnotation]() display the axis labels where the line is placed. The [ShowAxisLabel]() property is used to show or hide the axis’s label. This feature can be enabled by setting the [ShowAxisLabel]() property to true, as shown in the code snippet below:
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+    <chart:SfCartesianChart.Annotations>
+        <chart:VerticalLineAnnotation X1="2.5" ShowAxisLabel="True"/>
+        <chart:HorizontalLineAnnotation Y1="25" ShowAxisLabel="True"/>
+    </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %} 
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    var vertical = new VerticalLineAnnotation()
+    {
+        X1 = 2.5,
+        ShowAxisLabel= true
+    };
+
+    var horizontal = new HorizontalLineAnnotation()
+    {
+        Y1 = 25,
+        ShowAxisLabel= true
+    };
+
+    chart.Annotations.Add(vertical);
+    chart.Annotations.Add(horizontal);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation with axis label in MAUI Chart](Annotation_image/axis_label.png)
+
+## Axis label customization
+
+The default appearance of the axis label can also be customized using the [AxisLabelStyle]() property. The following styles are used to customize the axis label:
+
+The [ChartLabelStyle]() property allows you to customize the axis labels by changing the font family, font size, font attributes, and text color. It is the default style for the [AxisLabelStyle]() property. The following properties can be used to customize the axis label:
```

Comment:
Remove this as we discussed 
![image](https://github.com/syncfusion-content/maui-docs/assets/103025761/07f648da-66c8-4508-b1e4-8dcb23b6c1de)

Reply:
removed the content